openc3-cosmos-cmd-tlm-api/app/controllers/process_status_controller.rb

```diff
@@ -31,6 +31,8 @@ def show
     return unless authorization('system')
     if params[:id].downcase == 'all'
       render :json => @model_class.all(scope: params[:scope])
+    elsif params[:id].split('__').length > 1
+      render :json => @model_class.get(name: params[:id], scope: params[:scope])
```

Comment:
What are we checking for here?

Reply:
Process names have the form of hostname__pid